src/cli/commands/devices.ts

```diff
@@ -111,6 +111,23 @@ const command: CommandModule<GlobalOptions> = {
           await executeCommand("devices.delete", [{ device_id: argv.id }], argv)
         }
       )
+      .command(
+        "list device providers",
+        "list device providers by provider category (optional)",
+        (yargs) => {
+          return yargs.option("provider-category", {
+            describe: "filter by provider category",
+            type: "string",
+          })
+        },
+        async (argv) => {
+          await executeCommand(
+            "devices.listDeviceProviders",
+            [{ provider_category: argv.providerCategory }],
+            argv
+          )
+        }
+      )
```

Comment:
❤️ 

the cli hasn't gotten much love recently

Reply:
yeah I hope it's working properly lol. I explain in a comment below but when running `cli:dev`, `--*` parameters don't seem to be passed